SEQLinkage/Core.py

```diff
@@ -108,6 +113,23 @@ def load_fam_info(self,fam_pop_file):
     def load_ind_samples(self,ind_sample_file):
         pass
 
+    def get_regions(self,step=1000):
```

Comment:
this is a new function. Please document what it is for.

Reply:
add doc
This is a function to separate whole genome to regions with 1000 variants if annotation file provide and no reference gene file provide. you can think each region is a "gene".